gtfs/spec/en/reference.md

```diff
@@ -413,6 +419,85 @@ Trip planners normally calculate transfer points based on the relative proximity
+|  wheelchair_traversal_time | Optional | The **wheelchair_traversal_time** field specifies the time needed to go through the pathway in a wheelchair. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown travel time. |
+|   |  | * **-1** - Pathway not wheelchair accessible. |
+|   |  | * **(non-negative integer)** - Number of seconds needed to travers this pathway in a wheelchair. |
+|  ramp_slope | Optional | The **ramp_slope** field specifies the slope ratio of the pathway. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown slope. |
+|   |  | * **(float)** - Slope ratio of the pathway. |
+|   |  | Note: This field is allowed only for pathway_mode 1 (walkway) and 3 (travelator). |
+|  stair_count | Optional | The **stair_count** field specifies the number of stairs of the pathway. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown number of stairs. |
+|   |  | * **(non-negative integer)** - Total number of stairs of the pathway. |
+|   |  | Note: This field is allowed only for pathway_mode 2 (stairs) and 4 (escalator). |
+|   |  | Note: The approximation of 1 floor = 15 stairs (or 12 for an escalator) can be used to generate approximative values. |
+|  pathway_name | Optional | The **pathway_name** field contains the name of the pathway, if any. Please use a name that people will understand in the local and tourist vernacular. |
+|  pathway_code | Optional | The **pathway_code** field contains a short text or a number that uniquely identifies the pathway for passengers. The **pathway_code** can be the same as pathway_id if it is passenger-facing. This field should be left blank for pathway without a code presented to passengers. Example: Elevator “A”. |
+|  signposted_as | Optional | The **signposted_as** field helps to generate schematic directions like 'follow signs to <signposted as>' where <signposted as> part represents specific text on the signs that should remain unaffected by translation into other languages. |
+|  instructions | Optional | The **signposted_as** field indicates walking instructions for the pathway, such as “Walk downstairs to the station concourse”. |
+
+### levels.txt
+
+File: **Optional**
+
+This file allows a better modelling of the pedestrian walkways within a station by placing its locations on different levels.
+
+|  Field Name | Required | Details |
+|  ------ | ------ | ------ |
+|  level_id | **Required** | The **level_id** field contains an ID that identifies a level. The **level_id** is dataset unique. |
+|  level_index | **Required** | The **from_stop_id** field contains a numeric index of the level that indicates relative position of this level in relation to other levels (levels with higher indices are assumed to be located above levels with lower indices). Ground level should have index 0, with levels above ground indicated by positive indices and levels below ground by negative indices. |
```

Comment:
I think this is meant to be `level_index` here?

Reply:
Fixed. My bad.